HelloWorldVue/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think we should not push the editor's configuration and other related information on Github.